styleguide/components/41-organisms/image-collections/image-gallery/image-gallery--multiple.twig

```diff
@@ -1,4 +1,4 @@
-<div class="image-gallery--wrapper multiple">
+<div class="image-gallery--wrapper multiple{% if type %} image-gallery--{{ type }}{% endif %}">
```

Comment:
I'd put the space before the {% if... %} and not inside it.

Reply:
Debatable... If we'd place the space before the if-statement, `<div class="image-gallery--wrapper multiple ">` will be rendered in the DOM for a image gallery with no type specified. And that's kind of weird with the trailing space, don't you think?